src/App.tsx

```diff
@@ -1,61 +1,36 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { AddTodoForm } from './components/AddTodoForm';
+
+import { Todo } from './types/Todo';
+
+import { getUserById } from './utils/getUserById';
+
+const neededTodos = todosFromServer.map((todo: Todo) => {
```

Comment:
will better. And u can move this function to separate directory - `utils`
```suggestion
const preparedTodos/visibleTodos = todosFromServer.map((todo: Todo) => {
```